src/data/Tokenize.py

```diff
@@ -22,3 +22,10 @@ def train_tokenizer(filePath, maxVocabSize, maxSequenceLength):
     data = sequence.pad_sequences(sequences, maxlen=maxSequenceLength)
 
     return data, targets, tokenizer.word_index
+
+
+def tokenize_text(tokenizer_path, text):
+    with open(tokenizer_path) as handle:
```

Comment:
You open `tokenizer_path` here but do not use handle. Is this `with` statement even needed then? Doesn't `pickle.load` handle the file opening?

Reply:
Good question. You have good point. This is stack overflow code so I didnt pay much attention once it worked